app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,39 @@
+class QuestionsController < ApplicationController
+  def index
+    @questions = Question.all
+  end
+
+  def new
+    @question = Question.new
+  end
+
+  def create
+    @question = Question.new(params.require(:question).permit(:title, :body, :resolved))
```

Comment:
Cool, that works. Could you elaborate a bit on the use of the `private` keyword? Is the intention here to prevent another object from messing with the current object's params?